src/utils/index.ts

```diff
@@ -317,6 +318,16 @@ const getDefenderContract = (
   return contracts.find((a) => `${a.network}-${a.address}` === `${resource.network}-${resource.address}`);
 };
 
+const parseMonitorAbi = (abi: AbiType | undefined) => {
+  // Because the way AbiType is typed (string | string[]), a list with 1 string item is interpreted as a string rather than string[]
+  // Therefore, JSON.parse may fail if the string is not a valid JSON
+  try {
```

Comment:
could been if statement instead of try as well
```
function isJsonString(str) {
    try {
        JSON.parse(str);
    } catch (e) {
        return false;
    }
    return true;
}